tests/unit/models/generation/test_vllm_generation.py

```diff
@@ -41,7 +43,7 @@
         "name": model_name,
     },
     "dtype": "bfloat16",
-    "max_new_tokens": 5,  # Small number of tokens for testing
+    "max_new_tokens": 16,  # Small number of tokens for testing
```

Comment:
can you change this locally in your test if you need it to be larger? We set it to a very small value so we can check for exact matches (to hopefully avoid small errors that build over time across hardware stacks as such)